react to-do/src/styles.css

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: black;
+  color: white;
+}
+
+input {
+  background-color: aqua;
+  border: .0625rem solid aqua;
+  border-radius: .3125rem;
+  width: 18.75rem;
+  padding: 0.5rem;
+  margin-right:.625rem;
+  font-size: 1rem;
+}
+
+button {
+  cursor: pointer;
+  border: .0625rem solid black;
```

Comment:
lets use meaningful name for css file and  put it in seperate folder: src>assets>css> todoList.css